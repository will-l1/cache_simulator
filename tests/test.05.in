8192
65536
200
22
10281
20424
30571
40757
50914
61159
5811
16124
26326
36626
46933
57205
1822
11975
22282
32444
42775
53001
63274
8050
18211
28479
38650
48828
58992
3730
13918
24110
34352
44583
54824
64977
9786
20013
30214
40444
50763
61047
5830
16037
26281
36484
46633
56825
1455
11663
21943
32168
42367
52686
62922
7684
17983
28294
38479
48776
58962
3714
13925
24182
34461
44717
54986
65251
10009
20163
30431
40753
50991
61178
5853
16056
26223
36494
46751
57082
1703
11878
22104
32421
42662
52831
63041
7673
17973
28181
38507
48672
58901
3527
13719
24030
34230
44398
54680
64950
9705
20032
30346
40526
50795
60970
5730
16043
26315
36455
46786
57092
1792
11952
22248
32586
42891
53114
63347
8105
18314
28572
38850
49149
59290
3986
14275
24463
34749
44996
55146
65459
10239
20563
30871
41148
51310
61461
6145
16394
26656
36877
47091
57276
1950
12199
22367
32657
42882
53196
63504
8308
18504
28664
38892
49055
59294
4041
14185
24478
34632
44843
55165
65360
10053
20278
30541
40873
51053
61391
6133
16288
26490
36689
46835
57063
1770
11958
22182
32517
42709
52899
63114
7846
18177
28427
38693
49022
59276
3913
14087
24259
34469
44758
54997
65320
10127
20393
30575
40902
51212
61511
6322
