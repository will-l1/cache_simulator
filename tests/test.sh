#!/bin/sh

echo ======================================================
echo ====================== TEST $1 ========================
echo ======================================================
if timeout 10 ./$2/$3 < tests/test.$1.in > tests/test.$1.out; then 
  if diff -w tests/test.$1.out tests/test.$1.expected > /dev/null; then
    echo PASSED
  else
    echo FAILED
    echo ======
    echo ____Your_output____ __Expected_Output___
   #echo +++++++++++++++++++ ++++++++++++++++++++
    diff -y -W 80 tests/test.$1.out tests/test.$1.expected 
    echo =====================
    echo Your Output:
    echo ++++++++++++++++
    cat tests/test.$1.out
    echo =====================
    echo Expected Output:
    echo ++++++++++++++++
    cat tests/test.$1.expected
    exit 1
  fi
elif [ $? -eq 124 ]; then
  echo TIMEOUT
  exit 1
else 
  echo Abnormal program termination: the program crashed
  echo Exit code $?
  exit 1
fi
