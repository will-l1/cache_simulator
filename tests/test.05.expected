Loaded value [0xb9cb17b29e5109d2] @ address 0x00000016
Loaded value [0xfe38610da9581637] @ address 0x00002829
Loaded value [0x5282aa432ee47c70] @ address 0x00004fc8
Loaded value [0x1f51932a4e0a3c16] @ address 0x0000776b
Loaded value [0x3679b846637ce83d] @ address 0x00009f35
Loaded value [0x30504c316093685d] @ address 0x0000c6e2
Loaded value [0xdfdcd944ae3d6d26] @ address 0x0000eee7
Loaded value [0x86a82c1831258a3e] @ address 0x000016b3
Loaded value [0x65f745217c9d8903] @ address 0x00003efc
Loaded value [0x526636a7c3dc49df] @ address 0x000066d6
Loaded value [0x6af92af207880c1c] @ address 0x00008f12
Loaded value [0x412c72ac7a1febcc] @ address 0x0000b755
Loaded value [0x302777ce3114cc75] @ address 0x0000df75
Loaded value [0xc3795768351f4130] @ address 0x0000071e
Loaded value [0xf98fc419bc3fe573] @ address 0x00002ec7
Loaded value [0x52af049bf1ef4c7f] @ address 0x0000570a
Loaded value [0x36cf52d0194c77d5] @ address 0x00007ebc
Loaded value [0x1426f478cb16c036] @ address 0x0000a717
Loaded value [0x6d138166af42098d] @ address 0x0000cf09
Loaded value [0xa0c3673c143e617d] @ address 0x0000f72a
Loaded value [0x05957410242c0761] @ address 0x00001f72
Loaded value [0xa21c8d3944ad1363] @ address 0x00004723
Loaded value [0xaed1ce0c1bf43b6c] @ address 0x00006f3f
Loaded value [0x11767068a62c6681] @ address 0x000096fa
Loaded value [0x06700c102b247b8a] @ address 0x0000bebc
Loaded value [0x7cf6f2b46d771297] @ address 0x0000e670
Loaded value [0xc4ed22f9679012f6] @ address 0x00000e92
Loaded value [0x333e64186fdb70c3] @ address 0x0000365e
Loaded value [0xb4351606e2a871ff] @ address 0x00005e2e
Loaded value [0x3e0a6a9837f6bdc7] @ address 0x00008630
Loaded value [0xd0f73317840d8f0a] @ address 0x0000ae27
Loaded value [0x5a0f027259cdf0a8] @ address 0x0000d628
Loaded value [0x3c58caa6317a0e17] @ address 0x0000fdd1
Loaded value [0xf27c3df7de7c63c2] @ address 0x0000263a
Loaded value [0xcf7e4ed8280869d0] @ address 0x00004e2d
Loaded value [0xb364678810b5544f] @ address 0x00007606
Loaded value [0x477701c5179f4946] @ address 0x00009dfc
Loaded value [0xdd9c8d6ad9f2590b] @ address 0x0000c64b
Loaded value [0x170a242fd3b60150] @ address 0x0000ee77
Loaded value [0x620f02179bdc1b03] @ address 0x000016c6
Loaded value [0xa5290dddea667b73] @ address 0x00003ea5
Loaded value [0xe0102c6c6e69af21] @ address 0x000066a9
Loaded value [0x3caa5cb17c7d26e7] @ address 0x00008e84
Loaded value [0x1d6d42509508103d] @ address 0x0000b629
Loaded value [0x127396d8ae076936] @ address 0x0000ddf9
Loaded value [0x17b46a185634e946] @ address 0x000005af
Loaded value [0xb9cca915fab01d34] @ address 0x00002d8f
Loaded value [0x97791e552c50235e] @ address 0x000055b7
Loaded value [0x1234f0a053747e74] @ address 0x00007da8
Loaded value [0x89134a64184e4801] @ address 0x0000a57f
Loaded value [0x1c3624d3b4305a76] @ address 0x0000cdce
Loaded value [0x38df116db7923827] @ address 0x0000f5ca
Loaded value [0x046692475cdca0e3] @ address 0x00001e04
Loaded value [0x3167cc3ad1526062] @ address 0x0000463f
Loaded value [0xd8bb7a922539333a] @ address 0x00006e86
Loaded value [0xb35e744f07ee7323] @ address 0x0000964f
Loaded value [0x01b509b5498a1c9f] @ address 0x0000be88
Loaded value [0x3fb945671d085d8a] @ address 0x0000e652
Loaded value [0x2a157b3fd7cb5a7f] @ address 0x00000e82
Loaded value [0x2b191520eb443433] @ address 0x00003665
Loaded value [0x9d6862ffd55a6c4b] @ address 0x00005e76
Loaded value [0xbc6c20c28b11c616] @ address 0x0000869d
Loaded value [0xf768a68b472432af] @ address 0x0000aead
Loaded value [0xda264979b1596b30] @ address 0x0000d6ca
Loaded value [0x2d5a2857593ba96a] @ address 0x0000fee3
Loaded value [0x272d3f2efd37244e] @ address 0x00002719
Loaded value [0x4970cb0280d15c4a] @ address 0x00004ec3
Loaded value [0xb194f6411d9c4329] @ address 0x000076df
Loaded value [0x637ce83df3039fe0] @ address 0x00009f31
Loaded value [0xc64a7528c14feb13] @ address 0x0000c72f
Loaded value [0xf74f2cf930fa74f6] @ address 0x0000eefa
Loaded value [0xfd416983c93f194a] @ address 0x000016dd
Loaded value [0x6fb8b7f45a4cf881] @ address 0x00003eb8
Loaded value [0xea9dbf3a1bf39d4d] @ address 0x0000666f
Loaded value [0x2dba5e72e0a81ea8] @ address 0x00008e8e
Loaded value [0xb2a9d14032221f3c] @ address 0x0000b69f
Loaded value [0x1259289728b8796d] @ address 0x0000defa
Loaded value [0x2574ff75ebaffa24] @ address 0x000006a7
Loaded value [0x74ab7cdacd5f3a8c] @ address 0x00002e66
Loaded value [0x4e9b87e94417d1da] @ address 0x00005658
Loaded value [0x4266f8566c5b99e9] @ address 0x00007ea5
Loaded value [0xf1ce0b1698640557] @ address 0x0000a6a6
Loaded value [0x26d3cf761e25c025] @ address 0x0000ce5f
Loaded value [0x527d9d86fb0c4f4a] @ address 0x0000f641
Loaded value [0x6f7047630b151b69] @ address 0x00001df9
Loaded value [0x755ba6795718b597] @ address 0x00004635
Loaded value [0x3d1b5775c045b987] @ address 0x00006e15
Loaded value [0x28641a08912dd951] @ address 0x0000966b
Loaded value [0x188f5cf01c68e8dd] @ address 0x0000be20
Loaded value [0x6d7507ad9004f8de] @ address 0x0000e615
Loaded value [0xa0e3df674d18af01] @ address 0x00000dc7
Loaded value [0x8f5c534bf88a511a] @ address 0x00003597
Loaded value [0xb9a417bf76aa1b24] @ address 0x00005dde
Loaded value [0xd937274749682039] @ address 0x000085b6
Loaded value [0x7dde46cec75a3be1] @ address 0x0000ad6e
Loaded value [0x48d7d44900a7170f] @ address 0x0000d598
Loaded value [0xade128176d3b27ae] @ address 0x0000fdb6
Loaded value [0x2b3f0ada9e2cdc66] @ address 0x000025e9
Loaded value [0x15a21dcf79471f1c] @ address 0x00004e40
Loaded value [0x154546c1b8ca6469] @ address 0x0000768a
Loaded value [0xbcd42a92aa0d2d2f] @ address 0x00009e4e
Loaded value [0xf4697a53b6f32c47] @ address 0x0000c66b
Loaded value [0xb93a03fe44dc656e] @ address 0x0000ee2a
Loaded value [0xfdbe010af727703f] @ address 0x00001662
Loaded value [0x04da7b1a5fbfa529] @ address 0x00003eab
Loaded value [0x76b6c91015b5f625] @ address 0x000066cb
Loaded value [0x358b702863812d37] @ address 0x00008e67
Loaded value [0x50a9626bc2a053a5] @ address 0x0000b6c2
Loaded value [0x08effca552f0718d] @ address 0x0000df04
Loaded value [0x45ea5da538860801] @ address 0x00000700
Loaded value [0x0edbae6921584257] @ address 0x00002eb0
Loaded value [0x6b14b2cc26ca5286] @ address 0x000056e8
Loaded value [0xd64a7d9df4ec7713] @ address 0x00007f4a
Loaded value [0x7d66e8351c181404] @ address 0x0000a78b
Loaded value [0x80c30c4a15a355c8] @ address 0x0000cf7a
Loaded value [0x80cd802783833735] @ address 0x0000f773
Loaded value [0xf7536a7699199862] @ address 0x00001fa9
Loaded value [0xbd007679ae7206b1] @ address 0x0000478a
Loaded value [0x0b00f2637a7d74a4] @ address 0x00006f9c
Loaded value [0x56fd53af8ead6906] @ address 0x000097c2
Loaded value [0x4e77719f3c7f289a] @ address 0x0000bffd
Loaded value [0xa7224b2b3b446d00] @ address 0x0000e79a
Loaded value [0xa2f1246a16232fef] @ address 0x00000f92
Loaded value [0x4f3a392bf15d5f6e] @ address 0x000037c3
Loaded value [0x898a7d352821a768] @ address 0x00005f8f
Loaded value [0x76396f684b5b8af8] @ address 0x000087bd
Loaded value [0x07fbcc7775d47b66] @ address 0x0000afc4
Loaded value [0xcd483130f3e50ed9] @ address 0x0000d76a
Loaded value [0xc4c6056700680430] @ address 0x0000ffb3
Loaded value [0x140f1a6d78f0ce75] @ address 0x000027ff
Loaded value [0x64416e481676fe57] @ address 0x00005053
Loaded value [0x0784c5790cbd935c] @ address 0x00007897
Loaded value [0x5b94d0903618b233] @ address 0x0000a0bc
Loaded value [0x0c435987aaf24a2a] @ address 0x0000c86e
Loaded value [0x500c1886222b3c11] @ address 0x0000f015
Loaded value [0x691f8e128826affa] @ address 0x00001801
Loaded value [0xc10625a57a17741e] @ address 0x0000400a
Loaded value [0x2a2c578539751eac] @ address 0x00006820
Loaded value [0x56663b6f064d0084] @ address 0x0000900d
Loaded value [0x3c0ae078c1f79502] @ address 0x0000b7f3
Loaded value [0x1913f38f12b9c4ca] @ address 0x0000dfbc
Loaded value [0xb78d7f541e5749cd] @ address 0x0000079e
Loaded value [0x93a29c19e6dd103d] @ address 0x00002fa7
Loaded value [0x7de7ac474dadd02b] @ address 0x0000575f
Loaded value [0x6b20eda8713b90a4] @ address 0x00007f91
Loaded value [0xfca256378cb45c50] @ address 0x0000a782
Loaded value [0x5c5122ae45292122] @ address 0x0000cfcc
Loaded value [0x03ade71f396afd44] @ address 0x0000f810
Loaded value [0x433de5ff075637f9] @ address 0x00002074
Loaded value [0x12b73cca135c257d] @ address 0x00004848
Loaded value [0x411eafda7cb06bc0] @ address 0x00006ff8
Loaded value [0x05af5de93eb98beb] @ address 0x000097ec
Loaded value [0xf16ed34b59cf1807] @ address 0x0000bf9f
Loaded value [0x07943ac8a7224b2b] @ address 0x0000e79e
Loaded value [0x4d1f35f04277f725] @ address 0x00000fc9
Loaded value [0x07622c1e567e42cd] @ address 0x00003769
Loaded value [0x78b14f7e19bf7301] @ address 0x00005f9e
Loaded value [0x6da1a777436436e5] @ address 0x00008748
Loaded value [0x19f48c2e6baabc22] @ address 0x0000af2b
Loaded value [0x015d6372896c65a7] @ address 0x0000d77d
Loaded value [0x3ee4dad74d2e6cd4] @ address 0x0000ff50
Loaded value [0xa8748c78ce78aac6] @ address 0x00002745
Loaded value [0x4b1a1aaf48a52129] @ address 0x00004f36
Loaded value [0x7c545272ad708cb4] @ address 0x0000774d
Loaded value [0xdd4d12af2941ccef] @ address 0x00009fa9
Loaded value [0xb807f457fb3dd931] @ address 0x0000c76d
Loaded value [0x1a342d650a356b76] @ address 0x0000efcf
Loaded value [0xaf1de7a04d2babf4] @ address 0x000017f5
Loaded value [0x01eb0a747e3cb2ef] @ address 0x00003fa0
Loaded value [0x526715a133483fb8] @ address 0x0000677a
Loaded value [0xb539e53a58239046] @ address 0x00008f51
Loaded value [0x4a45727a7c5d6e52] @ address 0x0000b6f3
Loaded value [0xefa74a138bbbdc6a] @ address 0x0000dee7
Loaded value [0xb3b21e3c58e73edc] @ address 0x000006ea
Loaded value [0xd2506814a1330edb] @ address 0x00002eb6
Loaded value [0xa23b7664651b4956] @ address 0x000056a6
Loaded value [0x9178dba52f5553e0] @ address 0x00007f05
Loaded value [0x8f09345eb8744d7d] @ address 0x0000a6d5
Loaded value [0xb0900c4ed66d753a] @ address 0x0000cea3
Loaded value [0x7dcb75796f1703ca] @ address 0x0000f68a
Loaded value [0x20f43dab03671b4b] @ address 0x00001ea6
Loaded value [0x805f1495e5686ced] @ address 0x00004701
Loaded value [0x022b987c57706f15] @ address 0x00006f0b
Loaded value [0x0f67887968334c2c] @ address 0x00009725
Loaded value [0x9973096a06734770] @ address 0x0000bf7e
Loaded value [0x40a247333b4c7695] @ address 0x0000e78c
Loaded value [0xe62a3bf84b4ddfe5] @ address 0x00000f49
Loaded value [0xc34e10214393f00c] @ address 0x00003707
Loaded value [0xc38a0d49d7935009] @ address 0x00005ec3
Loaded value [0x5311c30594066e9f] @ address 0x000086a5
Loaded value [0xaac8482e87744698] @ address 0x0000aed6
Loaded value [0xdf35dba8235aff6c] @ address 0x0000d6d5
Loaded value [0x48bd3a76705cc0c3] @ address 0x0000ff28
Loaded value [0xdcb79a1c73e4c77c] @ address 0x0000278f
Loaded value [0x6c2d3ba621680bdc] @ address 0x00004fa9
Loaded value [0xaa93c11e1f51932a] @ address 0x0000776f
Loaded value [0x814a3e2ee50c098c] @ address 0x00009fc6
Loaded value [0x2f4828a83354caa4] @ address 0x0000c80c
Loaded value [0x4ea167234e1c0c39] @ address 0x0000f047
Loaded value [0x56c76c3c3a8f148a] @ address 0x000018b2
